test/SqlExtension.IntegrationTests/SqlExtension.IntegrationTests.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <IsPackable>false</IsPackable>
```

Comment:
I think it would be better to have all the tests - both unit and integration - in a single project. Splitting them up doesn't seem like something we should need to do and will make code coverage stuff harder.

Reply:
Didn't realize you were running the tests on the solution, so the code coverage stuff shouldn't be a problem in that case. But I still think having all the tests in one place makes it easier to work with. 

(although the tests are failing now so you'll need to still do some work there to get them running in the pipeline first. Were you planning on doing that yourself? Maybe we can do that in a follow up PR and just disable them for now in this PR)

Reply:
Moved IntegrationTests to inside the Tests project, disabled all but the Ubuntu pipeline for now